### DISCUSSION TITLE
A1 environment

### DISCUSSION THREAD

**Discussion by Lookatator** [google/brax#364] [General]
Hi,

In the `README.md` you show a visualization of an A1 robot. I had a look in the code but did not find any implementation for that environment.

I was simply wondering if you had any implementation available somewhere, or if you are planning to release any implementation in the future.

**Comment by btaba** [comment]
Hi @Lookatator , check out this post from Erik https://github.com/google/brax/discussions/325#discussioncomment-5689237
